studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.utils.tsx

```diff
@@ -139,7 +140,7 @@ export const generateOtherRoutes = (ref?: string, project?: ProjectBase): Route[
           },
         ]
       : []),
-    ...(IS_PLATFORM
+    ...(SHOW_O11Y
```

Comment:
What happens when user enable logs locally but the logflare endpoint is unreachable? Does the UI fallback gracefully?

Reply:
ok, thanks for clarifying